### ISSUE TITLE
schedule/jobs?

### ISSUE THREAD

**Issue by Frozenlock** [juxt/tick#26]
Coming from https://github.com/jarohen/chime (after looking for a job scheduler).

It appears that the `schedule` namespace is in the deprecated directory; does it mean that `tick` is no longer able to act as a job scheduler?

**Comment by malcolmsparks** [comment]
Yes, until the job scheduler is upgraded to align to the latest API. Hold off using tick as a job scheduler for now.
